README.md

```diff
@@ -9,13 +9,13 @@ Azure Data Studio is a data management tool that enables you to work with SQL Se
 
 Platform | Link
 -- | --
-Windows User Installer | https://go.microsoft.com/fwlink/?linkid=2091882
-Windows System Installer | https://go.microsoft.com/fwlink/?linkid=2091491
-Windows ZIP | https://go.microsoft.com/fwlink/?linkid=2091490
-macOS ZIP | https://go.microsoft.com/fwlink/?linkid=2091489
-Linux TAR.GZ | https://go.microsoft.com/fwlink/?linkid=2091488
-Linux RPM | https://go.microsoft.com/fwlink/?linkid=2091487
-Linux DEB | https://go.microsoft.com/fwlink/?linkid=2092022
+Windows User Installer | https://go.microsoft.com/fwlink/?linkid=2094100
+Windows System Installer | https://go.microsoft.com/fwlink/?linkid=2094200
+Windows ZIP | https://go.microsoft.com/fwlink/?linkid=2094201
+macOS ZIP | https://go.microsoft.com/fwlink/?linkid=2094202
+Linux TAR.GZ | https://go.microsoft.com/fwlink/?linkid=2094101
+Linux RPM | https://go.microsoft.com/fwlink/?linkid=2094102
+Linux DEB | https://go.microsoft.com/fwlink/?linkid=2094203
 
 Go to our [download page](https://aka.ms/azuredatastudio) for more specific instructions.
```

Comment:
Could you add `Stevoni Corrected Keyboard Shortcut Execution Issue (#5480)` to the contributors list?  I'd like to keep the list up-to-date until we figure out the best way to call out contributors each month.